### ISSUE TITLE
Update pyyaml when 4.2 release is available

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#4022]
## Description

We got a GitHub security alert regarding a CVE in pyyaml: [CVE-2017-18342](https://nvd.nist.gov/vuln/detail/CVE-2017-18342). The fix, [merged in the pyyaml repo](https://github.com/yaml/pyyaml/pull/74), renames `safe_load` to `load` and `load` to `danger_load`. We're already using safe_load for this project, e.g.:  https://github.com/freedomofpress/securedrop/blob/0a901362b84a5378fba80e9cd0ffe4542bdcd598/install_files/ansible-base/inventory-dynamic#L46

However, we shouldn't update to latest yet due to the fact that the only 4.2.* versions available on PyPI are marked as pre-release - see https://github.com/yaml/pyyaml/issues/193 for the full details. 

tl;dr We should wait until pyyaml 4.2 is released and then update.

**Comment by emkll** [comment]
A stable version of PyYaml has been released with a fix to the issue: https://pypi.org/project/PyYAML/5.1/

**Comment by rmol** [comment]
@redshiftzero @emkll Molecule pins PyYAML==3.13, so pip-tools can't reconcile all the dependencies if we add 5.1:

**Comment by redshiftzero** [comment]
Ah, good find. I think we should wait for upstream to update here, issue on molecule's side to update PyYAML: https://github.com/ansible/molecule/issues/1806

**Comment by redshiftzero** [comment]
the file to watch on molecule's side is [here](https://github.com/ansible/molecule/blob/master/setup.cfg#L83)

**Comment by redshiftzero** [comment]
upstream PR: https://github.com/ansible/molecule/pull/2064

**Comment by conorsch** [comment]
Upstream has merged new pyyaml: https://github.com/ansible/molecule/blob/master/setup.cfg#L85

**Comment by redshiftzero** [comment]
I think we need to wait for 2.22 to be out before updating (don't think that commit made its way into a release)

**Comment by emkll** [comment]
closed via #4865
